mineral-ui/mineral-ui
# Update minimum React/ReactDOM peer dependency version

### Description

Update minimum React/ReactDOM peer dependency version

### Motivation and context

Additional testing revealed that mineral-ui is functional down to React `15.3.0`.

closes #353

### Screenshots, videos, or demo, if appropriate

https://353-minimum-react-version--mineral-ui.netlify.com/

### How to test

1. Try out different React versions using the new test script in https://github.com/mineral-ui/mineral-ui/pull/447
2. Also, create a local release, install it in a CRA app using 15.3.0, import/render some components and ensure everything works.

### Types of changes

- Bug fix (non-breaking change which fixes an issue)
- Other (provide details below)

Sort of a bug fix, sort of a chore.  But then again, aren't all bugs chores?  They sure feel like it anyway...

### Checklist

* [x] Rendering performance (initial load time & 60fps) and [perceived performance] - **[n/a]**(http://blog.teamtreehouse.com/perceived-performance) considered
